ui/src/router/edit/components/VersionComparisonView.js

```diff
@@ -46,6 +46,7 @@ export const VersionComparisonView = ({
 
       <EuiFlexItem>
         <StepActions
+          currentStep={1}
```

Comment:
The decision to show "Cancel" vs "Previous" buttons is done in the library component based on the step number ([ref](https://github.com/gojek/mlp/blob/main/ui/packages/lib/src/components/multi_steps_form/StepActions.js#L20)), as opposed to comparing the value of `onCancel`, as was being done in the Turing component.